src/LpBaseNode.cpp

```diff
@@ -374,18 +378,36 @@ bool LpBaseNode::make_openvslam_config(const sensor_msgs::msg::CameraInfo::Share
     configNode["Camera"]["p2"] = 0.0;
 
     if (m_isStereoCamera) {
-        configNode["Camera"]["focal_x_baseline"] = -msg->p[3];  // -p[0,3]
+        configNode["Camera"]["focal_x_baseline"] = -left_msg->p[3];  // -p[0,3]
     }
 
     configNode["Camera"]["fps"] = m_cameraFps;
-    configNode["Camera"]["cols"] = msg->width;
-    configNode["Camera"]["rows"] = msg->height;
+    configNode["Camera"]["cols"] = left_msg->width;
+    configNode["Camera"]["rows"] = left_msg->height;
 
     // Trying to get Camera.color_order
     if (!get_camera_color_order(configNode)) {
         return false;
     }
 
+    // Stereo rectification parameters
+    configNode["StereoRectifier"]["model"] = "perspective";
+    std::vector<double> Dl(std::begin(left_msg->d), std::end(left_msg->d));
+    std::vector<double> Rl(std::begin(left_msg->r), std::end(left_msg->r));
+    std::vector<double> Kl(std::begin(left_msg->k), std::end(left_msg->k));
```

Comment:
Better to be `const std::vector<>` to tell the compiler that these values could be optimized-out